fe/be-java-extensions/avro-scanner/src/main/java/org/apache/doris/avro/AvroColumnValue.java

```diff
@@ -0,0 +1,195 @@
+                byteBuffer.putInt((int) value);
+                break;
+            case BIGINT:
+                byteBuffer = ByteBuffer.allocate(8);
+                byteBuffer.putLong((long) value);
+                break;
+            case FLOAT:
+                byteBuffer = ByteBuffer.allocate(4);
+                byteBuffer.putFloat((float) value);
+                break;
+            case DOUBLE:
+                byteBuffer = ByteBuffer.allocate(8);
+                byteBuffer.putDouble((double) value);
+                break;
+            case CHAR:
+            case VARCHAR:
+            case STRING:
+            case BINARY:
+            default:
+                byteBuffer = ByteBuffer.allocate(value.toString().length());
+                valueBytes = ByteBuffer.allocate(value.toString().length())
+                        .put(value.toString().getBytes(StandardCharsets.UTF_8)).array();
+        }
+    }
+
+    private Object inspectObject() {
+        byteBuffer.flip();
+        switch (columnType.getType()) {
+            case BOOLEAN:
+                return byteBuffer.get() == 1;
+            case TINYINT:
+                return byteBuffer.get();
+            case SMALLINT:
+                return byteBuffer.getShort();
+            case INT:
+                return byteBuffer.getInt();
+            case BIGINT:
+                return byteBuffer.getLong();
+            case FLOAT:
+                return byteBuffer.getFloat();
+            case DOUBLE:
+                return byteBuffer.getDouble();
+            case CHAR:
+            case VARCHAR:
+            case STRING:
+                return new String(valueBytes, StandardCharsets.UTF_8);
+            case BINARY:
+                return valueBytes;
+            default:
+                return new Object();
+        }
+    }
+
+    @Override
+    public boolean isNull() {
+        return false;
+    }
+
+    @Override
+    public boolean getBoolean() {
+        return (boolean) inspectObject();
+    }
+
+    @Override
+    public byte getByte() {
+        return (byte) inspectObject();
+    }
+
+    @Override
+    public short getShort() {
+        return (short) inspectObject();
+    }
+
+    @Override
+    public int getInt() {
+        return (int) inspectObject();
+    }
+
+    @Override
+    public float getFloat() {
+        return (float) inspectObject();
+    }
+
+    @Override
+    public long getLong() {
+        return (long) inspectObject();
+    }
+
+    @Override
+    public double getDouble() {
+        return (double) inspectObject();
+    }
+
+    @Override
+    public BigInteger getBigInteger() {
+        return null;
+    }
+
+    @Override
+    public BigDecimal getDecimal() {
+        return (BigDecimal) inspectObject();
+    }
+
+    @Override
+    public String getString() {
+        return inspectObject().toString();
+    }
+
+    @Override
+    public LocalDate getDate() {
+        return LocalDate.now();
```

Comment:
`LocalDate.now()` is only used mock jni reader, please check this.

Reply:
Maybe it can be changed to return null here？
Avro does not have date type，The code will not run to here either.
what you think？